blocks/filter.py

```diff
@@ -96,6 +96,16 @@ class object contained in the list.
     """
     def __init__(self, roles=None, bricks=None, each_role=False, name=None,
                  name_regex=None, applications=None):
+        if bricks is not None and not isinstance(bricks, Brick):
```

Comment:
Sorry about the assert, bad habit I guess.

I agree with getting rid of the convention of accepting both `Brick` and a list of `Brick`, but the main issue here would not be solved by that: list of lists are accepted but ignored.

Let me know if you want me to get rid of the `Brick` convention.